### PR TITLE
(#1582) Update Collections Storybook

### LINKED ISSUE #1582: Update storybook structure for nci-collection
### `nci-collection` storybook structure
1. Default (folder)
    1. Default
    2. Media
    3. Calendar
2. Variants (folder)
    1. Condensed 

Other tasks:
* Clean up all of the regressions (redundant breakpoint scenarios) associated with this component
* Make sure scenarios still reflect all of the test cases that we have, but we do not need them as examples in the storybook structure
- [x] Add collection passthrough (exact copy of what we had before)
- [x] Remove USWDS test cases from NCIDS collection

> **ESTIMATE** TBD

### Additional details / screenshot
- [Storybook structure wiki](https://github.com/NCIOCPL/ncids/wiki/Storybook-Structure)

### Related Tickets
* Parent ticket: #1224

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1582

### CODE REVIEW

**Comment by andyvanavery31** [comment]
@karlikpj Please change the name of the variant from "Only Headers" to "Condensed" and remove the USWDS pass through test cases

**Review by andyvanavery31 — CHANGES_REQUESTED** [review]
Can you please update the "Headers Only" name to be "Condensed" in the "Variants" folder?

For the regression images, can you please investigate the following?

- For the calendar, I am seeing 2 changes occurring due to breakpoints:
  - The calendar icon changes size between tablet and tablet large (879px to 880px) but the `ncids_nci-collection_calendar_mobile_size_0_document_0_tablet-lg.png` shows the smaller icon, but tablet large should have the same icon size as desktop. Not sure if this is a naming issue of the file or if we need to grab a different image.
  - The text of the title and body text starts reducing the width of its wrapping inside the tablet breakpoint (I'm seeing it start at 788px/789px). Do we need all images of breakpoints smaller than where this begins, or is having just Mobile sufficient
- Can you update the names regression image files for consistency? I'll defer to your judgement on naming, but things I'm seeing are:
  - `ncids_usa-collection_calendar_0_document_0_mobile.png` vs `ncids_usa-collection_calendar_desktop_size_0_document_0_desktop.png` vs `ncids_usa-collection_caledar_mobile_size_0_document_0_tablet-lg.png` (for other stories they were all in the form of the first one)
  - `ncids_usa-collection_media_0_document_0_mobile.png` vs ncids_usa-collection_media_image_not_visible_0_document_0_tablet-lg.png` vs `ncids_usa-collection_media_image_visible_0_document_0_desktop.png` (I'm not sure what would be best, I like the first one, but having the indication of the media image showing vs not showing is nice because of the breakpoints)
- Can you update the name of the regression image for `ncids_usa-collection_only_headers_0_document_0_mobile.png` to `ncids_usa-collection_condensed_0_document_0_mobile.png`

**Comment by karlikpj** [comment]
@andyvanavery31 there is a miss size in a breakpoint or two from the original file / fixing that - and that should rename to the correct size/ test case names

**Review by andyvanavery31 — CHANGES_REQUESTED** [review]
Could you please update the Variants > Only Headers to Variants > Condensed? And ensure that the naming of the images reflects that change?

**Review by andyvanavery31 — APPROVED** [review]
Looks good to me!

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
> - [ ] Add collection passthrough (exact copy of what we had before)
> - [x] Remove USWDS test cases from NCIDS collection

These tasks on the ticket were not done. We can talk about it. @bryanpizzillo @sarinapadilla @andyvanavery31

File: testing/ncids-css-testing/stories/uswds-native/usa-collection/src/usa-collection.scenarios.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-collection passthough default link focus',
+		storyId: 'uswds-components-collection--default',
+		focusSelector: '.usa-link',
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/uswds-native/usa-collection/src/usa-collection.scenarios.js:49]: if there are no styles for usa-link in usa-collections, we do not need this many tests for usa-link. usa-link shoud be tested in usa-link.

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**